kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/Main.java

```diff
@@ -96,9 +98,11 @@
     jc.addCommand("schema", new SchemaCommand(console));
     jc.addCommand("info", new InfoCommand(console));
     jc.addCommand("show", new ShowRecordsCommand(console));
+    jc.addCommand("obj-schema", new ObjectSchemaCommand(console));
     jc.addCommand("csv-schema", new CSVSchemaCommand(console));
     jc.addCommand("csv-import", new CSVImportCommand(console));
-    jc.addCommand("obj-schema", new ObjectSchemaCommand(console));
+    jc.addCommand("json-schema", new JSONSchemaCommand(console));
+    jc.addCommand("json-import", new JSONImportCommand(console));
```

Comment:
What do you think about separating the top-level commands like this?

* kite-dataset - dataset operations: create, delete, update, show, copy
* kite-import - import for various formats: csv, json, inputformat, etc.
* kite-config - generate configuration: csv-schema, json-schema, flume, partitioning, mapping

Or maybe break kite-config into kite-config and kite-schema?